config/logging.go

```diff
@@ -9,11 +9,12 @@ import (
 )
 
 type LoggingConfig struct {
-	TestLogCollect *bool            `toml:"test_log_collect"`
-	RunId          *string          `toml:"run_id"`
-	Loki           *LokiConfig      `toml:"Loki"`
-	Grafana        *GrafanaConfig   `toml:"Grafana"`
-	LogStream      *LogStreamConfig `toml:"LogStream"`
+	TestLogCollect         *bool            `toml:"test_log_collect"`
+	ShowHTMLCoverageReport *bool            `toml:"show_html_coverage_report"` // Show reports with go coverage data
```

Comment:
can we make it optional by providing `omitempty`?

Reply:
Yes I would prefer it to be `omitempty` for all

Reply:
@AnieeG `toml:",omitempty"`?

Reply:
Right. `toml:",omitempty"`

Reply:
you mean `toml:"<CURRENT_NAME>,omitempty"`, right? :-)

Reply:
@Tofel I think `toml:",omitempty"`. I see this is how core sets it https://github.com/smartcontractkit/chainlink/blob/develop/core/config/toml/types.go#L43. @Tofel @AnieeG  pls confirm

Reply:
@Tofel has been following a convention of differentiating the names for tablenames and keyname under the table.  Like 

> [Network] is table name and `selected_networks` is a key name that's under that table.

If we decide to avoid doing that, skipping explicit names makes sense.

Reply:
@kalverra thanks for explaining it. I added `omitempty` in next commit